src/sys/windows/stdnet/mod.rs

```diff
@@ -0,0 +1,25 @@
+mod addr;
```

Comment:
Can you add some docs to this module that it's "port" of `std::os::unix::net` that should be added to std in the future.